Adyen/UI/Form/Items/Value/FormValueItem.swift

```diff
@@ -24,7 +24,7 @@ public protocol FormValueItemStyle: TintableStyle {
 open class FormValueItem<ValueType: Equatable, StyleType: FormValueItemStyle>: FormItem {
 
     /// :nodoc:
-    public var subitems: [FormItem] = []
+    public var subitems: [FormItem] { [] }
```

Comment:
why? :-D

Reply:
For override. `public var subitems: [FormItem] = []` is **{get set}**, but `public var subitems: [FormItem] { [] }` is **{get}**

Reply:
then its can be let, right?:
```
public let subitems: [FormItem] = []
```

Reply:
Not really, because than it become immutable and you can't override or change it, and whole idea of `CompoundItem` is that items could change asynchronously